up.go

```diff
@@ -409,17 +429,17 @@ func (v *BufView) DrawTo(region Region) {
 	}
 
 	lclip := false
-	drawch := func(x, y int, ch rune) {
+	drawch := func(x, y int, mainc rune, combc []rune) {
 		if x <= v.X && v.X != 0 {
-			x, ch = 0, '«'
+			x, mainc = 0, '«'
```

Comment:
I consider again and think that it's better way to use a simple sequence of characters instead of mainc and combc like drawText function as you recommended. So,  I made similar changes to `(v *BufView) DrawTo` as below.  

Sorry for a little bit large, could you review?

https://github.com/akavel/up/pull/42/commits/9bb3ce57b4bbf983b5d5dbe8e84657b14869225a